client/js/game.js

```diff
@@ -16054,7 +16054,9 @@ function Game() {
         } else T.draw(ctx,"cjq4",W*0.5+bgw*0.5-T.width("cjq4")*0.5-14,H*0.5-bgh*0.5,T.width("cjq4")*0.75,T.height("cjq4")*0.75); 
     }
     this.updateLoopEventRanking = function (leaderboardName) {
-        if (eventRanking==undefined) eventRanking={};
+        if (eventRanking==undefined) eventRanking={time: 0, ptime: 0};
+		if(Date.now() - eventRanking.time < 2*60*1000)
+			return false;
```

Comment:
Still not sure about this bit, maybe remove it for now and look how much of a diffrence the other changes make?

```suggestion
```

Reply:
Ok